azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -318,18 +335,15 @@ public void cleanUpDanglingYarnApplications() {
     try {
       logger.info("cleanUpDanglingYarnApplications start ");
 
-      Set<Integer> executionStoppedFlows = getExecutionStoppedFlows();
-      logger.info("Get executionStoppedFlows: " +
-          executionStoppedFlows.stream().map(Object::toString).collect(Collectors.joining(",")));
-
+      Set<Integer> recentlyTerminatedFlows = getRecentlyTerminatedFlows();
       // get those flows terminated but containers are still alive (failed to properly killed)
       Set<Integer> toBeCleanedContainers = getContainersOfTerminatedFlows();
       logger.info("Get terminatedContainers: " +
           toBeCleanedContainers.stream().map(Object::toString).collect(Collectors.joining(",")));
 
       // combine both sets
-      toBeCleanedContainers.addAll(executionStoppedFlows);
-      logger.info("Get the set of all executions: " +
+      toBeCleanedContainers.addAll(recentlyTerminatedFlows);
```

Comment:
Wondering, why is this called `toBeCleanedContainers`, I don't think the current logic confines this to containers which are pending cleanup, is that correct ?

Reply:
good call out, this piece of code was copied from another method and reuse the naming, will rename to `toBeCleanedFlows`